src/tests/ftest/util/ior_test_base.py

```diff
@@ -224,7 +244,7 @@ def check_subprocess_status(self, operation="write"):
                 self.job_manager.process, self.ior_cmd):
             self.fail("Exiting Test: Subprocess not running")
 
-    def run_ior(self, manager, processes, intercept=None, display_space=True,
+    def run_ior(self, manager, processes, intercept=None,  display_space=True,
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.